MLS.Agent.Tests/MLS.Agent.Tests.csproj

```diff
@@ -10,6 +10,12 @@
     <EmbeddedResource Remove="TestResults\**" />
     <None Remove="TestResults\**" />
   </ItemGroup>
+
+  <ItemGroup>
+    <None Include="TestAssets\**">
```

Comment:
Is there only one file in TestAssets? In my opinion, the test that uses this might be more readable if it used our DisposableDirectory abstraction to create a temp directory with a single file and verified that serving static files worked that way. Otherwise, we've added yet another file that gets copied around by our build...

Reply:
Thanks! I have updated to use the "DisposableDirectory"